### PR TITLE
Update virtualbox.profile

### PR DESCRIPTION
**Pull request by Neo00001**

### CODE REVIEW

File: etc/profile-m-z/virtualbox.profile [hunk 1]
```diff
@@ -14,19 +14,34 @@ noblacklist /usr/lib/virtualbox
...
 mkdir ${HOME}/VirtualBox VMs
 whitelist ${HOME}/.config/VirtualBox
 whitelist ${HOME}/VirtualBox VMs
 whitelist ${DOWNLOADS}
 include whitelist-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
 include whitelist-var-common.inc
 
-caps.keep net_raw,sys_admin,sys_nice
```
Comment (rusty-snake) [inline etc/profile-m-z/virtualbox.profile:29]: Sure this does not break? On which distros have you tested it?

File: etc/profile-m-z/virtualbox.profile [hunk 2]
```diff
@@ -14,19 +14,34 @@ noblacklist /usr/lib/virtualbox
...
 netfilter
 nodvd
+#nogroups
 notv
+shell none
+tracelog
+
+#disable-mnt
+private-cache
+private-etc alsa,asound.conf,ca-certificates,dconf,fonts,hostname,hosts,ld.so.cache,pulse,localtime,machine-id,conf.d,resolv.conf,ssl,X11
```
Comment (rusty-snake) [inline etc/profile-m-z/virtualbox.profile:44]: Add crypto-policies and pki.

File: etc/profile-m-z/virtualbox.profile [hunk 3]
```diff
@@ -14,19 +14,34 @@ noblacklist /usr/lib/virtualbox
...
+
+dbus-user none
```
Comment (rusty-snake) [inline etc/profile-m-z/virtualbox.profile:46]: This breaks things such as notifications and tray-icons, has virtualbox and (usefull) usage of such things?

Comment (Neo00001) [reply etc/profile-m-z/virtualbox.profile:46]: Virtualbox doesn't have such features.

File: etc/profile-m-z/virtualbox.profile (hunk 2 above)
Comment (Neo00001) [reply etc/profile-m-z/virtualbox.profile]: done

File: etc/profile-m-z/virtualbox.profile (hunk 1 above)
Comment (Neo00001) [reply etc/profile-m-z/virtualbox.profile:29]: I have tested on manjaro(KDE). Kernel version: 5.7.9. you can even use `caps.drop all` . Do you have any profile testing community or group?

**Comment by kortewegdevries** [comment]
Have you tested these? https://github.com/netblue30/firejail/issues/2868#issuecomment-518647630

**Comment by Neo00001** [comment]
> Have you tested these? [#2868 (comment)](https://github.com/netblue30/firejail/issues/2868#issuecomment-518647630)

I tested with NAT.  NAT is virtualbox's default.But it seems without `sys_admin` host-only network doesn't work. So,my suggestion would be rather than enabling `sys_admin` by default, use it in comment. `sys_admin` is overloaded.

**Comment by Neo00001** [comment]
@kortewegdevries  hey can you take a look into this [vmware](https://github.com/Neo00001/frjlprfls/blob/master/vmware) profile.

**Comment by rusty-snake** [comment]
Merged, Thanks.

File: etc/profile-m-z/virtualbox.profile (hunk 1 above)
Comment (Neo00001) [reply etc/profile-m-z/virtualbox.profile:29]: `caps.keep net_raw,sys_admin,sys_nice` 
it's needed to be in comment

**Pull request merged**